src/ray/gcs/gcs_server/gcs_node_manager.cc

```diff
@@ -40,12 +40,14 @@ void GcsNodeManager::HandleRegisterNode(const rpc::RegisterNodeRequest &request,
                                         rpc::SendReplyCallback send_reply_callback) {
   NodeID node_id = NodeID::FromBinary(request.node_info().node_id());
   RAY_LOG(INFO) << "Registering node info, node id = " << node_id
-                << ", address = " << request.node_info().node_manager_address();
+                << ", address = " << request.node_info().node_manager_address()
```

Comment:
cc @scv119 should we use ip as the default or node id? Any opinion?

Reply:
I think node id is already printed in most cases. Defaulting to ip address can provided more info in those cases.